src/replace-menu/util/ReplaceOptionsUtil.js

```diff
@@ -0,0 +1,7 @@
+import * as replaceOptions from 'bpmn-js/lib/features/replace/ReplaceOptions';
+
+const ALL_OPTIONS = Object.values(replaceOptions).flat();
+
+export function getReplaceOptions(element) {
+  return ALL_OPTIONS;
```

Comment:
There are duplicates in the list (e.g. for `replace-with-collapsed-subprocess`). It might screw up the order when we use all entries, although it is unlikely when we mostly use ServiceTasks which are only present in the `TASK` collection

Reply:
What I'm doing now is insert by previous / next ref, rather than the absolute index: https://github.com/bpmn-io/bpmn-js-connectors-extension/pull/35/commits/df7dbaa4b20e36c2654c53b0252b2e46f33c020a.

Reply:
If that approach has issues we can revert it again.

Reply:
Sounds great. 

Let's do it and consider what is not working a bug that needs fixing :).